include/aspect/particle/world.h

```diff
@@ -111,6 +111,18 @@ namespace aspect
         const Particles::ParticleHandler<dim> &
         get_particle_handler() const;
 
+        /**
+         * Copy the state of particle handler @p from_particle_handler into the
+         * particle handler @p to_particle_handler. This will copy
+         * all particles and properties and leave @p to_particle_handler
+         * as an identical copy of @p from_particle_handler, assuming it
+         * was set up by this particle world class. This means we assume
+         * @p from_particle_handler uses the same triangulation and
+         * particle properties as are used in this model.
+         */
+        void copy_particle_handler (const Particles::ParticleHandler<dim> &from_particle_handler,
+                                    Particles::ParticleHandler<dim> &to_particle_handler) const;
```

Comment:
At least in deal.II we list the destination first (vmult for example). I don't insist that you change it that way, but it might be more intuitive.

Reply:
No, we have input first, output last. vmult really is a historical mistake.

Reply:
Oh great. I always look up vmult() to know the correct order. :-)